guides/source/configuring.md

```diff
@@ -1642,6 +1642,18 @@ The default value depends on the `config.load_defaults` target version:
 | (original)            | `true`               |
 | 7.1                   | `false`              |
 
+#### `config.active_record.protocol_adapters`
+
+When using a URL to configure the database connection, this option provides a mapping from the protocol to the underlying
+database adapter. This means the environment can specify `DATABASE_URL=mysql://localhost/database` and Rails will map
+`mysql` to the `mysql2` adapter, but the application can also override these mappings:
```

Comment:
Should we specify the other default mappings here too? (`sqlite` => `sqlite3` and `postgres` => `postgresql`?)

Reply:
I thought about it, but I worry about docs getting out of date from code. My goal here wasn't to specify _all_ mappings but offer an example of what one would look like (and the most common one).

Reply:
Code comments:
<img width="911" alt="image" src="https://github.com/rails/rails/assets/84159/826eaff0-8268-4b7c-9ffd-fd352694cd23">

Reply:
Avoiding an exhaustive list makes sense, and agree this is currently the most common reason someone might want to override our default (even though MySQL is not the most common database). It doesn't particularly avoid watermeloning us, because we expect this default to change (and, worse [for documentation clarity], become `load_defaults`-version dependent), but that's probably fine. Perhaps throw in a "For example," to emphasise that point?

(Full disclosure I'm currently being lazy and not reading surrounding options' docs to factor in their precedents around phrasing etc.)

The "but" here feels a little bumpy to me: seems more appropriate if this was describing AR connection behaviour [_but_ you can customize it]; here we're specifically talking about the config option, so what it does when set _is the thing_.